package.json

```diff
@@ -11,29 +11,32 @@
     "url": "https://github.com/sponsors/tannerlinsley"
   },
   "type": "module",
-  "types": "build/legacy/index.d.ts",
-  "main": "build/legacy/index.cjs",
-  "module": "build/legacy/index.js",
+  "types": "src/index.d.ts",
+  "module": "src/index.js",
```

Comment:
I would suggest we don't add this. It's not used by Node, and all modern bundlers (which we will be using) also read `"exports"`.